tests/element_test_vectors.rs

```diff
@@ -534,77 +534,38 @@ mod native_element_tests {
 
         fn global_skip_list() -> SkipList {
             &[
```

Comment:
🗺️ Here I've removed the now-passing `bad` test files from the skip list and added some comments explaining what needs to be done to address the remaining ones.